src/main/java/net/neoforged/neoforge/client/gui/ConfigurationScreen.java

```diff
@@ -715,18 +763,20 @@ protected <T extends Enum<T>> Element createEnumValue(final String key, final Va
                             }));
         }
 
+        @Nullable
         protected Element createIntegerValue(final String key, final ValueSpec spec, final Supplier<Integer> source, final Consumer<Integer> target) {
             final Range<Integer> range = spec.getRange();
-            final Integer min = range != null ? range.getMin() : 0;
-            final Integer max = range != null ? range.getMax() : Integer.MAX_VALUE;
+            final int min = range != null ? range.getMin() : 0;
+            final int max = range != null ? range.getMax() : Integer.MAX_VALUE;
```

Comment:
Why is there a `null` check for `range` here? `ListValueSpec#getSizeRange()` is not nullable (and the field it's pulling from has a `requireNonNullElse` guard).

Reply:
![image](https://github.com/user-attachments/assets/4d084e7e-96f1-4ed0-9569-a162474b3b20)

Range should be null for everything that hasn't got one...

Reply:
PS: Oh, I see. you mixed up getRange() and getListRange(). getRange() is nullable, as only Numeric values can have ranges, but lists always have numeric sizes, so ListRange isn't.

Reply:
Oh wait. I was blind and didn't quite understand the difference between `ValueSpec#getRange()` and `ListValueSpec#getSizeRange()`. Ignore me, heh.